extensions/wikia/Email/EmailMobileBadges.class.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class EmailMobileBadges
+ *
+ * @package Email
+ */
+class EmailMobileBadges {
+
+	static private $badges = [
+		'en' => [
+			'ios' => 'App-store-badge-en.png',
+			'android' => 'Google-play-badge-en.png',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "android" 14 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)